### DISCUSSION TITLE
Would like to contribute, can someone give me some guidance?

### DISCUSSION THREAD

**Discussion by Boy19991** [melink14/rikaikun#1088] [General]
Hello, rikaikun users.

I have novice/intermediate php/javascript knowledge, and am a long-time user of rikaikun (I dream of making a rikaikun for hebrew, spanish and russian). 

Is there a dude around here to give some guidance? Like tell me exactly what feature to work on and maybe answer some questions about development.

Cheers for all the great work of everyone.

**Comment by melink14** [comment]
Hi again and thanks for offering to help.

This is a list of issues marked as good first issues:
https://github.com/melink14/rikaikun/issues?q=label%3A%22good+first+issue%22+is%3Aopen+sort%3Aupdated-desc

This is a list of easy issues (with a lot of overlap of course):
https://github.com/melink14/rikaikun/issues?q=label%3A%22easy%22+is%3Aopen+sort%3Aupdated-desc

To start out, make sure to read the contributing guide here: https://github.com/melink14/rikaikun/blob/main/CONTRIBUTING.md

Rikaikun is written in Typescript with a mix of CSS and HTML for rendering so your JS knowledge will be helpful but you'll also need to learn some of the quirks of typed languages.

Let me know if any of the issues in the lists above stand out or seem interesting to you!
